doc/quality_metrics_service.md

```diff
@@ -0,0 +1,25 @@
+# Content quality metrics in PAAS
+
+### Getting set up with PAAS
+
+First you will need to get a PAAS account, the instructions on how to do this can be found [here](https://docs.cloud.service.gov.uk/get_started.html#get-an-account)
+
+Next you will need to install the Cloud Foundry CLI, instructions on how to do this can be found [here](https://docs.cloud.service.gov.uk/get_started.html#set-up-command-line)
+
```

Comment:
What about replacing it with:

1. [Create a PAAS account](https://docs.cloud.service.gov.uk/get_started.html#get-an-account)
2. [Install the Cloud Foundry CLI](https://docs.cloud.service.gov.uk/get_started.html#set-up-command-line)